example-scripts/01-sample-script.py

```diff
@@ -0,0 +1,57 @@
+"""Module 1 - 
```

Comment:
The idea here is that we have a simple script that has a lot of repeated code. We then will turn this into a more modular / function based script which will be 02-sample-script.py